document_page_portal/security/document_page_portal_security.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <odoo>
+    <data noupdate="1">
+
+         <record id="knowledge_portal_document_page_rule" model="ir.rule">
+            <field name="name">Documents: portal users: portal or following</field>
+            <field name="model_id" ref="document_page.model_document_page"/>
+            <field name="domain_force">[
+            '|',
+                '&amp;',
+                    ('type', '=', 'content'),
+                    ('message_partner_ids', 'child_of', [user.partner_id.commercial_partner_id.id]),
```

Comment:
Just an idea: Add a field on content pages in order to make them public for all portal users. For example, You want your usage manual to be accessible from everywhere. What do you think?

Reply:
@etobella Done! I added a new field to define whether the document is public or not.